packages/astro/src/core/app/index.ts

```diff
@@ -296,8 +300,12 @@ export class App {
 		}
 		if (!routeData) {
 			routeData = this.match(request);
+			this.#logger.debug("router", "Astro matched the following route for "+ request.url);
```

Comment:
Just curious why in some calls you use concatenation and others several arguments?

Reply:
Readability, mostly